docs/guides/upgrading.rst

```diff
@@ -276,6 +277,11 @@ Notable changes in plugins:
  * search plugin no longer extends ``page/elements/header`` and instead extends ``page/elements/sidebar``
  * topbar menu items might now have a new parent item or be found in a different section
 
+Removed / changed language keys
+-------------------------------
+
+ * The language keys related to notifications about comments have changed 
```

Comment:
Perhaps create a new section for notifications and describe what needs to be updated: plugins that override the action should stop sending the notification to avoid dupes etc.